app/models/katello/glue/candlepin/pool.rb

```diff
@@ -161,8 +161,8 @@ def import_data(index_hosts_and_activation_keys = false)
           pool_attributes[:unmapped_guest] = true
         end
 
-        if subscription.try(:redhat?)
-          pool_attributes[:virt_who] = pool_attributes['virt_limit'] != "0" && pool_attributes['virt_limit'].present?
+        if pool_attributes.key?(:virt_limit)
```

Comment:
I would go a step further and remove this if  statement
```pool_attributes[:virt_who] = pool_attributes['virt_limit'] != "0" && pool_attributes['virt_limit'].present?```
is good enough and more readable